lang/gcc7/Portfile

```diff
@@ -40,7 +40,10 @@ checksums           rmd160  91d46ec088badec75f41a2ad2a0ba228a6715107 \
 # If it is, libgcc7 installs a full runtime, otherwise it only installs
 # what is missing from newer libgccX builds.
 # NOTE : The logic here must match that in the libgcc port.
-set isLastSupported [ expr ${os.major} < 10 ]
+# set isLastSupported [ expr ${os.major} < 10 ]
```

Comment:
the whole “isLastSupported” mechanism code can be removed from this Portfile now, right? It won’t ever be re-enabled here.

Of course, when we peg again, probably on libgcc14 for older systems, then we’ll have to add all this isLastSupported mechanism again to the one we peg at.

Reply:
Might as well get rid of the `isLastSupported` mechanism, noted. That'll be quite a big change though, so perhaps it'd be more prudent to do it in a separate follow-up PR once this has been merged?

Reply:
Done, removed from this file.